app/script/view_model/ConversationInputEmojiViewModel.js

```diff
@@ -49,10 +49,29 @@ z.ViewModel.ConversationInputEmojiViewModel = class ConversationInputEmojiViewMo
       $(event.currentTarget).addClass('selected');
     });
 
-    fetch('/image/emoji.tsv')
-      .then((response) => response.text())
-      .then((text) => {
-        this.emoji_dict = text.split('\n').filter((event) => event.length > 0);
+    fetch('/image/emoji.json')
+      .then((response) => response.json())
+      .then((json) => {
+        this.emoji_dict = Object.entries(json)
```

Comment:
`Object.entries()` is only supported on Chrome 54 and newer. Please replace.

https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Object/entries